src/frequenz/sdk/actor/power_distributing/request.py

```diff
@@ -2,22 +2,30 @@
 # Copyright © 2022 Frequenz Energy-as-a-Service GmbH
 """Definition of the user request."""
 
-from dataclasses import dataclass
-from typing import Set
+from __future__ import annotations
 
+import dataclasses
 
-@dataclass
+
+@dataclasses.dataclass
 class Request:
-    """Request from the user."""
+    """Request to set power to the `PowerDistributingActor`."""
 
-    # How much power to set
     power: int
-    # In which batteries the power should be set
-    batteries: Set[int]
-    # Timeout for the server to respond on the requests.
+    """The amount of power to be set."""
```

Comment:
What do you think about:
``` suggestion
    """The requested power in watts."""

Reply:
Thanks, I didn't know the unit and thought of finding out later and then forgot!